code/modules/projectiles/gun.dm

```diff
@@ -475,7 +475,7 @@
 		playsound(user, fire_sound_silenced, 15, 1, -5)
 	else
 		playsound(user, fire_sound, 60, 1)
-
+		/*
```

Comment:
Why keep it commented? Is this standard practice in this repo?

Reply:
kinda , and also to let anyone turn it back on if they get the idea to do so. for downstreams , etc.